src/Styleguide/Components/RelayPagination.tsx

```diff
@@ -0,0 +1,193 @@
+import { Sans } from "@artsy/palette"
+import { RelayPagination_pageCursors } from "__generated__/RelayPagination_pageCursors.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled, { css } from "styled-components"
+import { themeGet } from "styled-system"
+import { Arrow } from "Styleguide/Elements/Arrow"
+import { Flex } from "Styleguide/Elements/Flex"
+import { Responsive } from "Styleguide/Utils/Responsive"
+
+interface PaginationProps {
+  pageCursors: RelayPagination_pageCursors
+  onClick?: (cursor: string) => void
+  onNext?: () => void
+  onPrev?: () => void
+}
+
+export class Pagination extends React.Component<PaginationProps> {
```

Comment:
Is there a reason to name this file `RelayPagination`? My intuition says having it named the same as whatever the main export is would be more clear. I'm not sure the `Relay` part is needed in this case. I could be missing some context though because I've been out of the loop for a little bit.

Reply:
In terms of naming / structure this PR is a bit more accurate: https://github.com/artsy/reaction/pull/861/files#diff-12a311781e8e426fdd7a00bd086dc7fcR60